templates/donation/informations.html.twig

```diff
@@ -33,7 +33,7 @@
                 Toutes ces informations sont obligatoires pour que votre don soit validé par la commission des comptes.
             </div>
             <div class="l__row l__row--h-stretch b__nudge--bottom-tiny form__radio-button">
-                {{ form_widget(form.gender, { attr: { class: 'register__form__gender' } }) }}
+                {{ form_widget(form.gender, { attr: { class: 'register__form__gender civility' } }) }}
```

Comment:
you can use `b__nudge--bottom-small` 

https://github.com/EnMarche/en-marche.fr/blob/master/front/style/layout/_layout.scss#L355

Reply:
Nice catch, thx !